pyugrid/ugrid.py

```diff
@@ -881,7 +883,7 @@ def save_as_netcdf(self, filepath):
                 face_nodes.cf_role = "face_node_connectivity"
                 face_nodes.long_name = ("Maps every triangular face to "
                                         "its three corner nodes.")
-                face_nodes.start_index = 0
+                face_nodes.start_index = IND_DT(0)
```

Comment:
wow -- I have never had to think about dtypes for attributes, and it sure seems like a bug in the netcdf lib -- but sure, this makes sense.

Reply:
Ok, I have removed the commit for the “optionally required” attributes face_dimension and edge_dimension, although I still think that it would not harm to specify them.

So please merge the start_index issue.